hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -500,7 +503,7 @@ class Lifter(handlerPaths: Opt[HandlerPaths])(using State, Raise):
           mod.foreach(applyClsLikeBody)
       
       def isFun(d: Defn) = d match
-        case FunDefn(owner, sym, params, body) => true
+        case FunDefn(owner, sym, dSym, params, body) => true
```

Comment:
Let's not keep dumb destructuring code like this that only makes maintenance more difficult
```suggestion
        case _: FunDefn => true
```